src/modules/dashboard/components/Extensions/ExtensionCard.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { useParams } from 'react-router';
+import Card from '~core/Card';
+import Icon from '~core/Icon';
+import Heading from '~core/Heading';
+
+import styles from './ExtensionCard.css';
+
+import { ExtensionData } from '~data/staticData/extensionData';
+import Link from '~core/Link';
+
+interface Props {
+  extension: ExtensionData;
+}
+
+const ExtensionCard = ({ extension }: Props) => {
+  const { colonyName } = useParams();
+  return (
+    <div className={styles.main}>
+      <Link to={`/colony/${colonyName}/extensions/${extension.extensionId}`}>
+        <Card className={styles.card}>
```

Comment:
I'm assuming the status tags and the "Add" controls directly on the extension card got deprecated, right ?

![Screenshot from 2020-11-17 14-30-54](https://user-images.githubusercontent.com/1193222/99390645-acfdd100-28e1-11eb-9ad9-d30160f7645b.png)